### PR TITLE
v10.2: Permissions on Variants

### PR DESCRIPTION
**Pull request by sofietoft**
Updated a couple of articles to reflect the new feature: Permissions of Language variants.

### CODE REVIEW

File: Fundamentals/Backoffice/Variants/index.md [hunk 1]
```diff
@@ -23,30 +35,41 @@ To work with Language Variants you need to have more than one language enabled.
...
 
     ![Allowing variance on properties](images/varying-content_v10.png)
 
-Each property editor that does not allow variants will be greyed out and have the content of the default language.
+Each Property Editor that does not allow variants (an Invariant Property) will by default need to be unlocked in order to be edited. The lock exists to make it clear that this change will affect more languages, as the value of the invariant properties are shared between all variants on the website.
+
+![How an invariant property looks when it is locked](images/invariant-property-locked.png)
+
+:::note
+Whether or not the lock is enabled on the invarient properties depend on the `AllowEditInvariantFromNonDefault`.
```
Comment (eshanrnh) [inline Fundamentals/Backoffice/Variants/index.md:65]:
```suggestion
Whether or not the lock is enabled on the invariant properties depends on the `AllowEditInvariantFromNonDefault` setting in the `appsettings.json` file.
```

File: Fundamentals/Backoffice/Variants/index.md [hunk 2]
```diff
@@ -23,30 +35,41 @@ To work with Language Variants you need to have more than one language enabled.
...
 
-To read about how you render variant content in templates, check out the [rendering content section](../../Design/Rendering-Content/).
+On projects starting on Umbraco 10.2 or later versions the setting is `true` by default. Was the project upgraded to 10.2 or later, the setting will be `false`.
```
Comment (eshanrnh) [inline Fundamentals/Backoffice/Variants/index.md:67]:
```suggestion
For projects using version Umbraco 10.2 or later, the setting is `true`, by default. If the project is upgraded to version 10.2 or later, you will need to add the `AllowEditInvariantFromNonDefault` setting in the `appsettings.json` file.

**Review by eshanrnh — APPROVED** [review]
2 small suggestions 😃

**Review by Zeegaan — APPROVED** [review]
LGTM 👍

**Pull request merged**